modules/simple_forms_api/spec/models/vba_20_10207_spec.rb

```diff
@@ -3,6 +3,48 @@
 require 'rails_helper'
 
 RSpec.describe 'SimpleFormsApi::VBA2010207' do
+  describe 'zip_code_is_us_based' do
```

Comment:
Is it valuable to include a test or tests that give the context where none of these addresses are given? In other words, are we throwing errors or otherwise catching this?

Reply:
Great point! I will add those! 👍